From 59c2da047bd6b94a7f6d0270e79f606d7b3402bf Mon Sep 17 00:00:00 2001
From: Jan Engelhardt <jengelh@inai.de>
Date: Thu, 29 Jan 2015 10:46:40 +0100
Subject: [PATCH 08/57] mime_util: force text/x-suse-ymp to be downloaded

YMP files (YaST One Click Install) are plaintext XML, but also not
very interesting in themselves. Force them to be stored.
---
 components/mime_util/mime_util.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/components/mime_util/mime_util.cc b/components/mime_util/mime_util.cc
index f7e59e4..592662f 100644
--- a/components/mime_util/mime_util.cc
+++ b/components/mime_util/mime_util.cc
@@ -73,6 +73,7 @@ static const char* const kUnsupportedTextTypes[] = {
     "text/x-csv",
     "text/x-vcf",
     "text/rtf",
+    "text/x-suse-ymp",
     "text/comma-separated-values",
     "text/csv",
     "text/tab-separated-values",
-- 
2.6.6

